sklearn/cluster/mean_shift_.py

```diff
@@ -13,6 +13,10 @@
 #          Alexandre Gramfort <alexandre.gramfort@inria.fr>
 #          Gael Varoquaux <gael.varoquaux@normalesup.org>
 
+# Modified: Martino Sorbaro <martino.sorbaro@ed.ac.uk>
```

Comment:
you can just add yourself to the authors above. Don't add a comment on the change here. Add it to whatsnew.rst.